water-sa.csl

```diff
@@ -0,0 +1,634 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Water SA</title>
+    <id>http://www.zotero.org/styles/water-sa</id>
+    <link href="http://www.zotero.org/styles/water-sa" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa-no-ampersand" rel="template"/>
+    <link href="http://www.wrc.org.za/Pages/KH_WaterSA.aspx?dt=5&amp;ms=62" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <issn>0378-4738</issn>
+    <eissn>1816-7950</eissn>
+    <updated>2017-04-04T15:00:30+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editortranslator" form="short">
```

Comment:
add definition for term edition = edn 
`      <term name="edition"  form="short">`
`        <single>edn</single>`
`        <multiple>edn</multiple>`
`      </term>`
solution for erro No. 19)